packages/terra-form/docs/ChoiceField.md

```diff
@@ -0,0 +1,22 @@
+# Choice Field
+
+Choice field component for rendering radio inputs in standard field structure.
+
+## Usage
+
+```jsx
+import React from 'react';
+import { ChoiceField } from 'terra-form';
+
+<ChoiceField
+  label="What State do you live in?"
+  name="children_present"
+  value="children_present"
```

Comment:
I think the values used in the `name` and `value` props need updated, they don't seem to make sense in the ChoiceField example.